Description: Removes the empty classifier for the tcnative dependency since it breaks
 the Gradle dependencies resolution (seen with the projectreactor package)
Author: Emmanuel Bourg <ebourg@apache.org>
Forwarded: not-needed
--- a/pom.xml
+++ b/pom.xml
@@ -444,7 +444,6 @@
         <groupId>${project.groupId}</groupId>
         <artifactId>${tcnative.artifactId}</artifactId>
         <version>${tcnative.version}</version>
-        <classifier>${tcnative.classifier}</classifier>
         <scope>compile</scope>
         <optional>true</optional>
       </dependency>
--- a/handler/pom.xml
+++ b/handler/pom.xml
@@ -63,7 +63,6 @@
     <dependency>
       <groupId>${project.groupId}</groupId>
       <artifactId>${tcnative.artifactId}</artifactId>
-      <classifier>${tcnative.classifier}</classifier>
       <optional>true</optional>
     </dependency>
     <dependency>
--- a/transport-native-epoll/pom.xml
+++ b/transport-native-epoll/pom.xml
@@ -248,7 +248,6 @@
     <dependency>
       <groupId>${project.groupId}</groupId>
       <artifactId>${tcnative.artifactId}</artifactId>
-      <classifier>${tcnative.classifier}</classifier>
       <scope>test</scope>
     </dependency>
   </dependencies>
--- a/transport-native-kqueue/pom.xml
+++ b/transport-native-kqueue/pom.xml
@@ -401,7 +401,6 @@
     <dependency>
       <groupId>${project.groupId}</groupId>
       <artifactId>${tcnative.artifactId}</artifactId>
-      <classifier>${tcnative.classifier}</classifier>
       <scope>test</scope>
     </dependency>
   </dependencies>
